backend/core/urls.py

```
@@ -1,8 +1,17 @@
 from django.contrib import admin
 from django.urls import path
+from rest_framework_simplejwt.views import (
+    TokenObtainPairView,
+    TokenRefreshView
+)
 from .views import *
```

Ruff

`from .views import *` used; unable to detect undefined names

(F403)